src/armTemplates/resources/functionApp.ts

```diff
@@ -1,14 +1,52 @@
 import { ArmParameter, ArmParameters, ArmParamType, ArmResourceTemplate, ArmResourceTemplateGenerator, DefaultArmParams } from "../../models/armTemplates";
-import { FunctionAppConfig, ServerlessAzureConfig, SupportedRuntimeLanguage } from "../../models/serverless";
+import { FunctionAppConfig, ServerlessAzureConfig, SupportedRuntimeLanguage, FunctionAppOS } from "../../models/serverless";
 import { AzureNamingService, AzureNamingServiceOptions } from "../../services/namingService";
+import configConstants from "../../config";
 
 interface FunctionAppParams extends DefaultArmParams {
+  /**
+   * Name of function app
+   */
   functionAppName: ArmParameter;
+  /**
```

Comment:
nit: new line before comments for readability, especially for text to speech reader

Reply:
Hmm, not sure. We haven't done that before, and we don't have new lines between the members of other interfaces. I don't know if a jsdoc comment should change that